project/Dependencies.scala

```diff
@@ -58,7 +58,7 @@ object Dependencies {
   val macwire = "com.softwaremill.macwire" %% "macros" % "2.5.7" % "provided"
   val mockito = "org.mockito" % "mockito-all" % "1.10.19" % Test
   val paClient = "com.gu" %% "pa-client" % "7.0.7"
-  val playGoogleAuth = "com.gu.play-googleauth" %% "play-v30" % "4.0.0"
+  val panDomainAuth = "com.gu" %% "pan-domain-auth-play_3-0" % "3.1.0"
   val playSecretRotation = "com.gu.play-secret-rotation" %% "play-v30" % "7.1.0"
   val playSecretRotationAwsSdk = "com.gu.play-secret-rotation" %% "aws-parameterstore-sdk-v1" % "7.1.0"
```

Comment:
Note that deleting `common/app/conf/GoogleAuth.scala` in this PR means there's no remaining use of `com.gu.play-secret-rotation` in the project, so these two dependencies can be removed too!

I'm not sure what the rotation policy is on the panda keys, I do wonder how often they are rotated?!

Reply:
great spot, thanks @rtyley 

I'll look into the panda key rotation.